test/src/specs/sync/chain_forks.rs

```diff
@@ -509,18 +507,32 @@ impl Spec for ForksContainSameTransactions {
 
         // Build `chain1`, contain the target `transaction`, with length = 61
         {
-            sleep(Duration::from_secs(2));
```

Comment:
`sleep(2s)` makes the timestamp of blocks are different, to avoid `chain1 == chain0`. I agree that `sleep` is tricky. We can change the `block.nonce` or `block.timestamp` manually to archive the intention.

Reply:
Why it is 2 seconds?

Reply:
`chain1` and `chain2` have different height and the `transaction` is in the different height block, they are different.

Reply:
I change my previous description: to ensure that `chain1[1] != chain2[1]`.

Reply:
updated.